public/docs/ts/latest/guide/attribute-directives.jade

```diff
@@ -64,9 +64,9 @@ a#write-directive
 
 :marked
   ### Write the directive code
-  Create a new project folder (`attribute-directives`) and follow the steps in [QuickStart](../quickstart.html).
+  Follow the [setup](setup.html) instructions for creating a new project
+  named <ngio-ex path= <ngio-ex path="attribute-directives"></ngio-ex>.
```

Comment:
There are two successive `<ngio-ex path= ` on this line. You can also write this more simply as `<span ngio-ex>attribute-directives</span>` if you want.